### PR TITLE
Add escape key as another shortcut to exit current view.

### LINKED ISSUE #6085: UI: Navigation: Additionally allow Esc-key to navigate back.
#### Problem Description
First time using mitmproxy instead of mitmweb:
I intuitively tried navigating back to the flow list (initial view) after entering the view of a single flow using <kbd>Esc</kbd>.

#### Proposal
Additionally to <kbd>q</kbd> allow to use <kbd>Esc</kbd> to navigate back.

### PR DESCRIPTION
**Pull request by sujaldev**
#### Description

The escape key can now be used to exit the current view, alongside the pre-existing `q` key. Fixes #6085.

#### Checklist

 - [x] I have updated tests where applicable.
 - [x] I have added an entry to the CHANGELOG.

### CODE REVIEW

**Comment by sujaldev** [comment]
Should I add the fix for #6086 in this PR itself?

**Comment by Prinzhorn** [comment]
> Should I add the fix for #6086 in this PR itself?

No, please don't mix unrelated things in PRs.

File: mitmproxy/tools/console/keymap.py [hunk 1]
```diff
@@ -71,9 +72,7 @@ def sortkey(self):
 class Keymap:
     def __init__(self, master):
         self.executor = commandexecutor.CommandExecutor(master)
-        self.keys: dict[str, dict[str, Binding]] = {}
-        for c in Contexts:
-            self.keys[c] = {}
+        self.keys: dict[str, dict[str, Binding]] = defaultdict(dict)
```
Comment (mhils) [inline mitmproxy/tools/console/keymap.py:75]: What is this change about?

Comment (sujaldev) [reply mitmproxy/tools/console/keymap.py:75]: I thought it'd make it more readable. Should I revert this?

Comment (mhils) [reply mitmproxy/tools/console/keymap.py:75]: Looking at bit closer this is probably fine :)

**Review by mhils — APPROVED** [review]
Thanks! 🍰

**Pull request merged**